src/components/Editor.jsx

```diff
@@ -43,11 +43,7 @@ const makeMarker = () => {
   return marker;
 };
 
-const pushValueToCodeMirror = _.once((value, codeMirrorInst) => {
-  if (!codeMirrorInst.getValue() && value) {
-    codeMirrorInst.setValue(value);
-  }
-});
+const pushValueToCodeMirror = (value, codeMirrorInst) => codeMirrorInst.setValue(value);
```

Comment:
> I also removed the call to window.location when a snippet is loaded via a user's snippet list. This way we do not have to reload the whole app and make unnecessary AJAX calls, resulting in a faster snippet loading experience.

This means that the value may need to be pushed more than once in a session, because multiple snippets can now be loaded in a single session.